Description: relax rustls dependency
 As debian only has rustls-0.18 packaged, depend on that version
 instead of 0.19.0
Author: Jan Niehusmann <jan@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Last-Update: 2021-05-12

--- rust-rustls-native-certs-0.5.0.orig/Cargo.toml
+++ rust-rustls-native-certs-0.5.0/Cargo.toml
@@ -22,7 +22,7 @@ categories = ["network-programming", "cr
 license = "Apache-2.0/ISC/MIT"
 repository = "https://github.com/ctz/rustls-native-certs"
 [dependencies.rustls]
-version = "0.19.0"
+version = "0.18.0"
 optional = true
 [dev-dependencies.ring]
 version = "0.16.5"
